xml/sle_update_background.xml

```diff
@@ -13,7 +13,7 @@
   <abstract>
    <para>
     This chapter provides background information on terminology, &suse; product
-    lifecycles and Service Pack releases, and recommended upgrade policies.
+    lifecycles fand Service Pack releases, and recommended upgrade policies.
```

Comment:
Hi Daria, this one looks fishy to me - maybe you accidentally pressed a key (or wanted to correct something else)...

Reply:
Sorry! That is very strange though - I can see in the file itself that I last touched it 12 months ago...